docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+```
+peer lifecycle chaincode checkcommitreadiness --channelID mychannel --name fabcar --version 1.0 --init-required --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --output json
+```
+
+The command will produce a JSON map that display if a channel member member has approved the parameters that were specified in the `checkcommitreadiness` command:
+```json
+    {
+            "Approvals": {
+                    "Org1MSP": true,
+                    "Org2MSP": true
+            }
+    }
+```
+
+Since both channel members have approved the same parameters, the chaincode definition can be committed to the channel. Issue the following command to commit the chaincode definition to the channel as the Org2 admin:
+```
+peer lifecycle chaincode commit -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name fabcar --version 1.0 --sequence 1 --init-required --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+```
+
+The commit command also needs to be submitted by an organization admin. Note that the transaction includes the address of the ordering node and uses the `--peerAddresses` flag to target peers in Org1 and Org2. The `commit` transaction needs to query the peers of channel members that have approved the chaincode definition. The transaction is then submitted to the ordering service to be added to a block and distributed to the channel, where the peers validate that a majority of organizations have approved the chaincode definition. The `peer lifecycle chaincode commit` command will wait for the validations from the peer before returning a response.
+
+You can use the ``peer lifecycle chaincode querycommitted`` command to confirm that the chaincode definition has been committed to the channel.
+```
+peer lifecycle chaincode querycommitted --channelID mychannel --name fabcar --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+If the chaincode was successful committed to the channel, the `querycommitted` command will return the sequence and version of the chaincode definition:
+```
+Committed chaincode definition for chaincode 'fabcar' on channel 'mychannel':
+Version: 1, Sequence: 1, Endorsement Plugin: escc, Validation Plugin: vscc, Approvals: [Org1MSP: true, Org2MSP: true]
+```
```

Comment:
I'm reminded here that nowhere have we explained the sequence number in cc lifecycle. I think there is general confusion around how that is used and might be worth including in the tutorial somewhere?

Reply:
Good catch! added "The `--sequence` parameter is an integer that keeps track of the number of a the chaincode has been defined or updated. Because the chaincode is being first deployed, the sequence number is 1. When you next upgrade the chaincode, the sequence number will be incremented to 2."